frontend/src/components/DatabaseBackup/utils.ts

```diff
@@ -0,0 +1,60 @@
+import { BackupPlanPolicySchedule, BackupSetting } from "@/types";
+
+export type BackupSettingEdit = Pick<
+  BackupSetting,
+  "enabled" | "dayOfWeek" | "hour" | "retentionPeriodTs"
+>;
+
+export const PLAN_SCHEDULES: BackupPlanPolicySchedule[] = [
+  "UNSET",
+  "WEEKLY",
+  "DAILY",
+];
+
+export const AVAILABLE_DAYS_OF_WEEK = [0, 1, 2, 3, 4, 5, 6];
+export const AVAILABLE_HOURS_OF_DAY = [0, 1, 2, 3, 4, 5, 6];
```

Comment:
This may not be the same for services targeting different time zones.

Reply:
This is defined as local TZ for selection. And will be converted to UTC when selected.

Reply:
SG.